jetty-slf4j-impl/src/test/java/org/eclipse/jetty/logging/JettyLoggerTest.java

```diff
@@ -526,7 +526,7 @@ public void testToString()
         assertThat("Logger.toString", log.toString(), is("JettyLogger:xxx:LEVEL=ERROR"));
 
         log.setLevel(JettyLogger.OFF);
-        assertThat("Logger.toString", log.toString(), is("JettyLogger:xxx:LEVEL=OFF"));
+        assertThat("Logger.toString", log.toString(), is("JettyLogger:xxx:LEVEL=ERROR"));
```

Comment:
What?
OFF != ERROR

Reply:
In #4831, "OFF" is converted to 999 and 999 is converted to Level.ERROR. So, uhm, why don't we get rid of OFF and just call it error so it conforms to SLF4J?